src/content/docs/new-relic-solutions/tutorials/add-tables.mdx

```diff
@@ -0,0 +1,263 @@
+
+Before you can use `EntityTitleTableRowCell` and `MetricTableRowCell`, you have to add them to the import statement first.
+
+```jsx lineNumbers=false copy=false
+import {
+    EntityTitleTableRowCell,
+    MetricTableRowCell,
+    ... /* All previous components */
+} from 'nr1';
+
+```
+
+</Step>
+<Step>
+
+Update your table rows by replacing the first and second `TableRowCell`s with entity and metric cells.
+
+Notice that `EntityTitleTableRowCell` are self-closing tags.
+
+```jsx lineNumbers=false
+{
+  ({ item }) => (
+    <TableRow>
+      <EntityTitleTableRowCell value={item} />
+      <MetricTableRowCell
+        type={MetricTableRowCell.TYPE.APDEX}
+        value={item.value}
+      />
+      <TableRowCell>{item.company}</TableRowCell>
+      <TableRowCell>{item.team}</TableRowCell>
+      <TableRowCell>{item.commit}</TableRowCell>
+    </TableRow>
+  );
+}
+```
+
+</Step>
+<Step>
+
+Time to give your table a second look: The cell components you've added take care of properly formatting the data.
+
+</Step>
+<Step>
+
+Tables are great, but interactive tables can be better: As a last update, you are going to allow users to act on each data row.
+
+Add the `_getActions()` method to your `index.js` file, right before `_getItems()`.
+
+As you may have guessed from the code, `_getActions()` spawns an alert box when you click **Team** or **Commit** cells.
+
+```jsx lineNumbers=false
+_getActions() {
+    return [
+        {
+            label: 'Alert Team',
+            iconType: TableRow.ACTIONS_ICON_TYPE.INTERFACE__OPERATIONS__ALERT,
+            onClick: (evt, { item, index }) => {
+                alert(`Alert Team: ${item.team}`);
+            },
+        },
+        {
+            label: 'Rollback Version',
+            iconType: TableRow.ACTIONS_ICON_TYPE.INTERFACE__OPERATIONS__UNDO,
+            onClick: (evt, { item, index }) => {
+                alert(`Rollback from: ${item.commit}`);
+            },
+        },
+    ];
+}
+
+```
+
+</Step>
+<Step>
+
+Find the `TableRow` component in your `return` statement and point the `actions` property to `_getActions()`.
+
+The `TableRow` actions property defines a set of actions that appear when the user hovers over a table row. Actions have a mandatory text and an `onClick` callback, but can also display an icon or be disabled if needed.
+
+```jsx lineNumbers=false
+<TableRow actions={this._getActions()}>
+```
+
+</Step>
+<Step>
+
+Go back to your application and try hovering over any of the rows: Notice how the two available actions appear. When you click them, a function triggers with the selected row data as an argument, and an alert displays in your browser.
+
+</Step>
+</Steps>
+
+## Next steps [#next]
+
+You've built a table into a New Relic application, using components to format data automatically and provide contextual actions. Well done!
+
+Keep exploring the `Table` components, their properties, and how to use them, in our SDK documentation.
```

Comment:
Where should this link to?

Reply:
It doesn't exist on our docs site yet. Let's leave it for now.